q4.txt

```diff
@@ -0,0 +1,20 @@
+javascript
+a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];
+var i=0;
+
+while(i  <a.length){
+      a.reverse();
```

Comment:
@saif99ss the question was to revers every other element, here you are reversing the sequence for all elements